library/asset/sprite/src/asset-sprite.cpp

```
@@ -0,0 +1,638 @@
+    .spritesheet = info.spritesheet,
+    .data = {},
+    .dataType = info.dataType,
+  };
+  switch (info.dataType) {
+    case PuleAssetSpriteLibraryDataType_singleSprite:
+      library.data.singleSprite = info.data.singleSprite;
+    case PuleAssetSpriteLibraryDataType_animationLibrary:
+      library.data.animationLibrary.animations.resize(
+        info.data.animationLibrary.animationCount
+      );
+      memcpy(
```

cppcheck (2.10-2)

[error] Using 'memcpy' on struct that contains a 'std

(memsetClass)